isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -2282,6 +2344,8 @@ namespace Isis {
    * @param label Pvl label to initialize from
    */
   void Cube::initCoreFromLabel(const Pvl &label) {
+    initialize();
+    initLabelState(label);
```

Comment:
why were these not needed before?

Reply:
This was a result of adding the initCoreFromGdal. As of right now it doesn't populate all of the values you need from the GDALDataset. Also not a bad idea to initialize all of the cube defaults before creating the cube from the label